app/assets/stylesheets/helpers/_emergency-banner-notifications.scss

```diff
@@ -1,113 +1,67 @@
 #emergency-banner-notification {
+
   background-color: $panel-colour;
-  border-top: 2px solid white;
-  padding: 0.5em 1em;
+  color: $white;
+  border-top: 2px solid $white;
+  padding: $gutter-half 0;
 
   @include media(tablet) {
-    padding: 0;
+    padding: $gutter 0;
   }
 
   div {
-    @include core-19;
-    color: #fff;
-    margin: 0 auto;
-    padding:0.5em 0;
-    max-width: 1020px;
-    overflow: hidden;
+    @extend %site-width-container;
+  }
+
+  h1 {
+    @include bold-24;
+    margin-bottom: 0;
 
-    @include ie-lte(7) {
-      width: 956px;
+    .homepage & {
+      @include bold-48;
+      @include media(tablet) {
+        margin-bottom: $gutter-two-thirds;
+      }
     }
   }
 
   p {
-    padding: 0;
-    margin: 0;
+    @include core-19;
+    color: $white;
+    margin-top: 0;
+    margin-bottom: $gutter-two-thirds;
 
     @include media(tablet) {
-      padding: 0 30px 0 30px;
+      width: percentage(2/3);
```

Comment:
Rather than using percentages widths under breakpoints you could consider using `max-width` with `em`s.

Since here you're trying to limit line length for legibility if you did something like the following it had a direct relationship with the amount of characters:

We did something similar on short-text descriptions https://github.com/alphagov/government-frontend/pull/323